squarelet/users/viewsets.py

```diff
@@ -110,6 +112,23 @@ def get_object(self):
             return super().get_object()
 
 
+class PressPassUserMembershipViewSet(
```

Comment:
This should also be in the organization app

Reply:
This has been moved as well.